.gitignore

```diff
@@ -22,3 +22,4 @@ earth_enterprise/tutorial/FusionTutorial/KML
 /*.files
 /*.includes
 
+.vscode/
```

Comment:
Shouldn't this be in the top level .gitignore like before??

Reply:
I made the change so regardless of which directory you load as your root in vscode, the project will ignore its contents.

I must have opened the source at earthenterprise/earth_enterprise, and saw the extra folder in my untracked files.

This change avoids the issue in all scenarios (including root).